cloudevents/languages/zh-CN/formats/avro-turbo-format.md

```diff
@@ -0,0 +1,6 @@
+# Avro Turbo Event Format for CloudEvents - Version 1.0.3-wip
+
+本文档尚未被翻译，请先阅读英文[原版文档](../../../formats/avro-tubro-format.md) 。
```

Comment:
tubro - typo

Reply:
Maybe it should be called tubro?

Reply:
your figures do seem to want to do that :-)